app/models/vehicle.rb

```diff
@@ -1,3 +1,7 @@
 class Vehicle < ApplicationRecord
-  has_many :reservations
+  has_many :reservations, dependant: :destroy
+
+  validates :description, presence: true, length: { maximum: 10_000 }
```

Comment:
[OPTIONAL] Should this be 10.000?

Reply:
This is a rubocop linter correction, Style/NumericLiterals: Use underscores(_) as a thousands separator and separate every 3 digits with them.

Thanks for the revision. 👍🏼